bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: opendatahub-operator
   operators.operatorframework.io.bundle.channels.v1: fast
-  operators.operatorframework.io.bundle.channel.default.v1: fast
+  operators.operatorframework.io.bundle.channel.default.v1: rolling
```

Comment:
so we set default channel for both v1 and v2 on rolling?

Reply:
Yes, else it defaults to fast channel, as v2>v1